src/Simulation/Simulators.Tests/TestProjects/QCIExe/ClassicallyControlledSupportTests.qs

```diff
@@ -0,0 +1,732 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
+namespace Microsoft.Quantum.Simulation.Testing.QCI.ClassicallyControlledSupportTests {
+
+    operation SubOp1() : Unit { }
+    operation SubOp2() : Unit { }
+    operation SubOp3() : Unit { }
+
+    operation SubOpCA1() : Unit is Ctl + Adj { }
+    operation SubOpCA2() : Unit is Ctl + Adj { }
+    operation SubOpCA3() : Unit is Ctl + Adj { }
+    
+    operation BasicLift() : Unit {
+        let r = Zero;
+        if (r == Zero) {
+            SubOp1();
+            SubOp2();
+            SubOp3();
+            let temp = 4;
+            using (q = Qubit()) {
+                let temp2 = q;
+            }
+        }
+    }
+
+    operation LiftLoops() : Unit {
+        let r = Zero;

```

Comment:
Same in this file; if you could just copy over the adaptions I mentioned for Honeywell, that would be great.